pkg/pager/model.go

```
@@ -0,0 +1,381 @@
+	m.statusBarView(b)
+	return b.String()
+}
+
+func max(a, b int) int {
+	if a > b {
+		return a
+	}
+	return b
+}
+
+func (m model) statusBarView(b *strings.Builder) {
```

golangci-lint (1.64.8)

[error] function-length: maximum number of lines per function exceeded; max 60 but got 63

(revive)